packages/apps/transfer/src/pages/faucet/existing/notification.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Button,
+  INotificationProps,
+  Notification,
+  NotificationBody,
+  NotificationFooter,
+  styled,
+  SystemIcons,
+} from '@kadena/react-components';
+
+import { RequestStatus } from '.';
+
+import React, {
+  FC,
+  MouseEventHandler,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+
+type FormNotificationProps = Pick<INotificationProps, 'title'> & {
+  status: RequestStatus;
+  body?: string;
+};
+
+const statusToColorMapping: Record<RequestStatus, INotificationProps['color']> =
+  {
+    failed: 'negative',
+    pending: 'primary',
+    succeeded: 'positive',
+    'not started': 'default',
```

Comment:
Since I also integrated this into my tracker page, these colors don't seem right according to the design files.

Reply:
The colors are defined in the Notification component coming from `@kadena/react-components`, so it would then need to be changed there. Is that what you mean?

Reply:
Indeed, but since you're not changing those files in this PR maybe we should leave it for another PR

Reply:
Agreed 👍